src/core/index.js

```diff
@@ -5,15 +5,10 @@ let is = require( '../is' );
 let Promise = require( '../promise' );
 let define = require( '../define' );
 
-let Core = function( opts ){
-  let cy = this;
-
-  opts = util.extend( {}, opts );
-
-  let container = opts.container;
-
-  // allow for passing a wrapped jquery object
-  // e.g. cytoscape({ container: $('#cy') })
+let coreInit = function( cy, opts ){
```

Comment:
I think `coreInit` is doing too much.  Probably `mount()` only needs a subset of what's going on here.  For example, re-mounting would destroy all the existing elements...